lib/injection.rb

```diff
@@ -0,0 +1,19 @@
+class Injection
+
+
+	def number
+		@number ||= []
+	end
+
+
+	def add(number)
+		number.reduce(:+)
+	end
+	
+
+	def multiply(number)
+		number.reduce(1,:*)
+	end
+
+
+end
```

Comment:
Check out how inject operates [here](http://ruby-doc.org/core-2.2.0/Enumerable.html#method-i-inject) - the technique you'll want to look at is how to use `yield` and how to update an accumulator as you move through an array